.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": [["@babel/plugin-proposal-nullish-coalescing-operator"]]
```

Comment:
I assume these changes are due to the waterfall PR?

I'll ignore what I already seen in previous PRs.

Reply:
oh yes, I'll rebase this branch now that we merged the other PR to main, sorry about the mess